README.md

```diff
@@ -0,0 +1,18 @@
+# handbook
+Handbook for interacting with the DANDI archive.
+
+## DANDI Style Sheet
+
+Follow the guidelines below when creating and revising text in the DANDI Handbook:
+
+*	**dandi-** repositories (hyphenate, dandi-archive, dandi-cli, *not* dandischema; dandisets is an exception because that is a
+complete word, i.e. it is not using dandi as a prefix to a noun)
+*	**dandiset** (**not** Dandiset)
```

Comment:
i'll need to think a bit about this one. i think this should be capitalized.

Reply:
I'm fine with consistently capitalizing "Dandiset"

Reply:
now it's  **Dandiset** (**not** dandiset)`